ui/src/shared/components/tables/TableCell.tsx

```diff
@@ -137,8 +137,12 @@ class TableCell extends PureComponent<Props> {
     return this.isFirstRow && this.isFirstCol
   }
 
-  private get isNumerical(): boolean {
-    return !isNaN(Number.parseFloat(this.props.data))
+  private get isTimestamp(): boolean {
+    return this.props.dataType === 'dateTime:RFC3339'
+  }
+
+  private get isNaN(): boolean {
+    return isNaN(Number(this.props.data))
```

Comment:
Passing in `null` has now changed. Previously, using Number.parseFloat(null) would return `NaN` and ultimately the expression is NaN

Number(null) returns 0, and ultimately the expression is **not** NaN

This looks like an improvement, because we _do_ want to handle null values. Just wanted to make sure it was intentional.

Reply:
@TCL735 it was intentional in terms of my understanding of how the UI should handle UI values, but it was not explicit in terms of what the criteria should have been. @sjwang90 what're your thoughts on the way we should handle `null` values?